### ISSUE TITLE
Printing *readtable* signaled an error

### ISSUE THREAD

**Issue by drmeister** [drmeister/clasp#189]
```
> *readtable*
../../src/core/exceptions.cc:133 af_signalSimpleError  caught because signal-simple-error is not installed yet
COMMON-LISP:PROGRAM-ERROR
end must be nil or [0,16)

af_signalSimpleError
../../src/core/foundation.cc:241 dbg_hook(...) was called

"#  8/b@0x2b72cab67da1        exceptions.cc   839/0   (SIGNAL-SIMPLE-ERROR PROGRAM-ERROR NIL \"end must be nil or [0,16)\" NIL )
"
"The following restarts are available:"
"ABORT      a    Abort to REPL"
Frame-8-Dbg[1]>
```

stassats pointed it out and I fixed this.

**Comment by Shinmera** [comment]
Where did you fix this? It's still there on e18cb701e1f5ad40bebfbede9ed903adbb1f6aad

**Comment by Shinmera** [comment]
Fixed on a3a0dcc321929fe7e0bcb664dc53b701c8179ae2
```
> *readtable*

#<COMMON-LISP:READTABLE :case :UPCASE> 
```
